jobs/sendEmailAlertNoFill.php

```diff
@@ -74,80 +72,43 @@ function sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template) {
-        foreach($criticalWarnedUsers as $u)
-            $criticalWarnedPeople = $criticalWarnedPeople . $u . "\r\n";
-        $lastWarnedPeople = "";
-        foreach($lastWarnedUsers as $u)
-            $lastWarnedPeople = $lastWarnedPeople . $u . "\r\n";
-
-        $message = file_get_contents($NO_FILL_TEMPLATE_MANAGERS);
-        $message = str_replace("###NO_FILL_DAYS_TRIGGER_WARNING###",$NO_FILL_DAYS_TRIGGER_WARNING,$message);
-        $message = str_replace("###NO_FILL_DAYS_TRIGGER_CRITICAL###",$NO_FILL_DAYS_TRIGGER_CRITICAL,$message);
-        $message = str_replace("###WARNED_PEOPLE###",$warnedPeople,$message);
-        $message = str_replace("###CRITICAL_PEOPLE###",$criticalWarnedPeople,$message);
-        $message = str_replace("###LAST_PEOPLE###",$lastWarnedPeople,$message);
-
-        mail($to, $subject, $message, $headers);
-    }
-
     $today = new DateTime(date('Y-m-d'));
     $initDate = new DateTime($today->format('Y').'-01-01');
 
     $users = UsersFacade::GetAllUsers();
-    $warnedUsers = [];
-    $criticalWarnedUsers = [];
-    $lastWarnedUsers = [];
-    $pingedUsers = [];
+    $excludedUsers = [];
 
     foreach ($users as $user) {
         $groups = $user->getGroups();
-        foreach ($groups as $group) {
-            if ($group->getName() == $ALL_USERS_GROUP) {
-                $report = UsersFacade::ExtraHoursReport($initDate, $today, $user);
-                $login = $user->getLogin();
-                $email = $login . "@" . $NO_FILL_EMAIL_DOMAIN;
-                $from = $NO_FILL_EMAIL_FROM;
-                $lastTaskDate = $report[1][$user->getLogin()]["last_task_date"];
-                $difference = $lastTaskDate->diff($today);
-                $difference = $difference->format('%a');
-                if ($difference >= $NO_FILL_DAYS_TRIGGER_LAST) {
-                    array_push($lastWarnedUsers, $login);
-                    $cc = $NO_FILL_CC_LAST;
-                    $subject = $NO_FILL_SUBJECT_LAST;
-                    $template = $NO_FILL_TEMPLATE_LAST;
-                    sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template);
-                } elseif ($difference >= $NO_FILL_DAYS_TRIGGER_CRITICAL) {
-                    array_push($criticalWarnedUsers, $login);
-                    $cc = $NO_FILL_CC_CRITICAL;
-                    $subject = $NO_FILL_SUBJECT_CRITICAL;
-                    $template = $NO_FILL_TEMPLATE_CRITICAL;
-                    sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template);
-                } elseif ($difference >= $NO_FILL_DAYS_TRIGGER_WARNING) {
-                    array_push($warnedUsers, $login);
-                    $cc = $NO_FILL_CC_WARNING;
-                    $subject = $NO_FILL_SUBJECT_WARNING;
-                    $template = $NO_FILL_TEMPLATE_WARNING;
-                    sendEmail($email, $lastTaskDate, $from, $cc, $subject, $template);
+        if (in_array($user->getLogin(), $excludedUsers))
+            continue;
+        else {
```

Comment:
A couple of nits here:
* If you use `continue` , you don't need the else branch. Or you could have written the condition as `!in_array` with no else branch.
* I'd have used curly braces to reduce the ambiguity.